test/system/120-load.bats

```diff
@@ -80,26 +80,94 @@ verify_iid_and_name() {
 
 @test "podman image scp transfer" {
     skip_if_remote "only applicable under local podman"
+    if is_ubuntu; then
```

Comment:
ubuntu gave me some issues on my first go as well, I can circle back in a separate PR and try to fix this

Reply:
my PR no longer uses machinectl to execute the commands but does use it to log in which is an issue... I am looking into how runuser can be used instead in the rootful form (which is all that matters in this case since rootless just uses an exec.Command(podman...))